src/ppx/geometry.cpp

```diff
@@ -1057,11 +1057,14 @@ uint32_t Geometry::GetVertexCount() const
 
 const Geometry::Buffer* Geometry::GetVertexBuffer(uint32_t index) const
```

Comment:
If we never return nullptr, would it be better to return Buffer& instead of Buffer*. I am always worried about possible nullptr when i see pointers.  (maybe not necessarily in this change since there might be many other places using this function)

Reply:
Noted, I agree that it should not happen in this PR though since the function is used elsewhere.